app/javascript/tests/public_health/query/AdvancedFilter.test.js

```diff
@@ -71,6 +71,10 @@ function checkStatementRender(wrapper, contents, index) {
         expect(statement.find('.advanced-filter-number-options').find('option').at(index).text()).toEqual(numberOptionValuesText[Number(index)]);
         expect(statement.find('.advanced-filter-number-options').find('option').at(index).prop('value')).toEqual(value);
       });
+      if (contents.filterOption.support_blank) {
+        expect(statement.find('.advanced-filter-number-options').find('option').at(numberOptionValues.length).text()).toEqual('');
+        expect(statement.find('.advanced-filter-number-options').find('option').at(numberOptionValues.length).prop('value')).toEqual('');
+      }
       expect(statement.find('.advanced-filter-number-input').exists()).toBe(true);
```

Comment:
this should fail if `support_blank` is true so this will need to be updated (same with line 69).  I'd also update the number filter mock to have support blank set so that it is tested

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1252/commits/00b46c7e5d22ccb24f7f0a01ec951b89f46bca12